feat: adds log page for builds

cdrage opened the pull request:
feat: adds log page for builds

### What does this PR do?

* Adds a log action button that will show the logs in real-time in a
  separate page
* Automatically refreshes / appends to the output
* Uses terminal settings from Podman Desktop to match what the user has
  setup.

### Screenshot / video of UI

<!-- If this PR is changing UI, please include
screenshots or screencasts showing the difference -->


https://github.com/user-attachments/assets/d440a86a-dee7-46cf-9aef-84aa1e3298d9


https://github.com/user-attachments/assets/05d6b51f-80c3-4717-8b40-dff1df61ea49



### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop
repository (or from another issue tracker). -->

Closes https://github.com/containers/podman-desktop-extension-bootc/issues/677

### How to test this PR?

1. Start a build
2. Click the "logs" button on the dashboard
3. Watch logs propagate

<!-- Please explain steps to reproduce -->

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

deboer-tim reviewed (changes requested):
Nice addition, just some minor feedback.

cdrage commented:
@deboer-tim updated!

* Removed the console logs... had them there for debugging and forgot to remove
* Renamed the get configuration value function

deboer-tim reviewed (approved):
Thanks for the changes. Just need to fix the linting but otherwise looks good. 👍🏼
